### PR TITLE
Add simple example how to test Golang actions locally

### PR DESCRIPTION
**Pull request by skywalkeretw**
Add a new example that simulates the action running locally for test purposes.

### CODE REVIEW

File: examples/EXAMPLES.md [hunk 1]
```diff
@@ -42,4 +42,5 @@ Available examples:
 - [Golang action with a package](package-main) main is `main.Main` invoking a `hello.Hello` and a test
 - [Golang action with a module](module-main) main is `main.Main` using a dependency `github.com/rs/zerolog`
 - [Standalone Golang Action](standalone) implements the ActionLoop directly in go
+- [Test Golang Action Localy](standalone-simple) execute Golang Action localy
```
Comment (rabbah) [inline examples/EXAMPLES.md:45]:
```suggestion
- [Test Golang Action Localy](standalone-simple) execute Golang Action locally
```

**Review by sciabarracom — COMMENTED** [review]
You basically put the main in the same file as the action itself, as such it can compile locally but it cannot be deployed on production as you will have a conflict on main. I would put the stub actionloop in a separate file, with a build tag to compile only locally. The stub would also require to add some of the mandatory extra values in the request args, such as apihost etc.

**Review by mrutkows — APPROVED** [review]
Thanks

**Review by mrutkows — CHANGES_REQUESTED** [review]
Truthfully, I had not idea that the Go runtime had this many examples.  I would ask that we better surface and differentiate the types of examples as even reading "example.md" I have no idea from the descriptions what makes each one different. 

The current descriptions seems like it uses "insider" terminology and assumes more intimate knowledge of the Action proxy.  

I would also change the top-level README to NOT link to example.md, but instead to the examples sub-directory so that you can actually see them by name; otherwise, you should rename "examples.md"->"README.md" within the examples subdirectory (as linking to examples.md) obscures the actual code and files it is meant to document). The actual directory/file names are themselves descriptive and somewhat self-documenting).

I would add a description of how to use a Makefile in that new README.md as well

**Comment by sciabarracom** [comment]
I still do not see a way to do a separate compilation of the "pseudo-main". So this will allow you to build a local action but not use it as a true action for the runtime.  Not sure we need this example it would be more confusing than useful.

**Pull request closed without merging**